python/cudf/cudf/tests/test_csv.py

```diff
@@ -17,7 +17,7 @@
 
 import cudf
 from cudf import read_csv
-from cudf.core._compat import PANDAS_GE_200, PANDAS_LT_140
+from cudf.core._compat import PANDAS_GE_200
```

Comment:
We will be able to assume `PANDAS_GE_200` is always true. Let's refactor to reflect that assumption of truth.

Reply:
Oh. Just saw https://github.com/rapidsai/cudf/pull/14910#issuecomment-1915586453.